lib/js/dom.nim

```diff
@@ -832,7 +853,7 @@ type
     FourthButton = 8,
     FifthButton = 16
 
-  # https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent
+  ## https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent
```

Comment:
doc comment at wrong place, generates wrong html:

![image](https://user-images.githubusercontent.com/2194784/76378774-8f687880-630b-11ea-8d36-f64886c7a58c.png)

* these doc comments need to be moved so the render properly
* these should display as clickable links eg like this:
```
Nim's standard library only covers the basics, check
out `<https://nimble.directory/>`_ for a list of 3rd party packages.
```

Reply:
Sorry I don't know what to do in this case:

```nim
  ## `https://developer.mozilla.org/en-US/docs/Web/API/Blob`_
  Blob* = ref BlobObj
  BlobObj {.importc.} = object of RootObj
    size*: int
    `type`*: cstring
```

Most of the cases involve double objects.

Do I put them on both? 

```nim
  Blob* = ref BlobObj ## `https://developer.mozilla.org/en-US/docs/Web/API/Blob`_  
  BlobObj {.importc.} = object of RootObj
    ## `https://developer.mozilla.org/en-US/docs/Web/API/Blob`_
    size*: int
    `type`*: cstring
```

Or on one? Do you have a preference on which one?

```nim
  Blob* = ref BlobObj 
  BlobObj {.importc.} = object of RootObj
    ## `https://developer.mozilla.org/en-US/docs/Web/API/Blob`_
    size*: int
    `type`*: cstring
```

Thank you for the quick response, I just want to get the commit to the state you are happy with.

Reply:
the doc comment should be attached to the thing that's exported:

```nim
  Blob* = ref BlobObj ## https://developer.mozilla.org/en-US/docs/Web/API/Blob
  BlobObj {.importc.} = object of RootObj
    size*: int
    `type`*: cstring
```
(when in doubt just run `nim doc` to see the output)

more generally, even if both were exported, I'd prefer to attach it to the one that's most likely to be used (the ref type) and not repeat the doc comment; docs should be DRY just like code (otherwise you always wonder what subtle difference in doc comment you missed)

Reply:
I made everything match now:
![image](https://user-images.githubusercontent.com/125948/76382586-744f3600-6316-11ea-888c-65749fbfd0e3.png)